libs/vm_modules/include/vm_modules/math/math.hpp

```diff
@@ -35,6 +37,36 @@ fetch::math::meta::IfIsMath<T, T> Abs(fetch::vm::VM *, T const &a)
   return x;
 }
 
+template <typename T, typename R = void>
+using IfIsNormalSignedInteger = meta::EnableIf<meta::IsSignedInteger<T> && sizeof(T) >= 4, R>;
+
+template <typename T, typename R = void>
+using IfIsSmallSignedInteger = meta::EnableIf<meta::IsSignedInteger<T> && sizeof(T) < 4, R>;
+
+template <typename T>
+meta::EnableIf<sizeof(T) < 4, int32_t> ToAtLeastInt(T const &value)
+{
+  return static_cast<int32_t>(value);
+}
+
+template <typename T>
+IfIsSmallSignedInteger<T, T> IntegerAbs(fetch::vm::VM *, T const &value)
+{
+  return static_cast<T>(std::abs(value));
+}
+
+template <typename T>
+IfIsNormalSignedInteger<T, T> IntegerAbs(fetch::vm::VM *, T const &value)
+{
+  return std::abs(value);
+}
+
+template <typename T>
+meta::IfIsUnsignedInteger<T, T> IntegerAbs(fetch::vm::VM *, T const &value)
```

Comment:
Could the `IntegerAbs` templates be incorporated into `math/standard_functions/abs.hpp`?

Reply:
@ejfitzgerald